### DISCUSSION TITLE
An exception occurred applying plugin request [id: 'nebula.netflixoss', version: '9.2.2']

### DISCUSSION THREAD

**Discussion by AditiRitam1209** [Netflix/conductor#2195] [Q&A] [answered]
I m getting  An exception occurred applying plugin request [id: 'nebula.netflixoss', version: '9.2.2'] when I am trying to setup conductor on my local machine

**Comment by maheshyaddanapudi** [comment]
Hi,

Are you building using a local Gradle installation or with the gradlew included in the code base ? Reason for asking this is, the Gradle installation in local might have a version which is not compatible for Conductor Server build.

I tried the following, using the gradlew and it worked.

1) git clone 
2) cd conductor
3) ./gradlew build -x test (This will build and place the JAR under server/build/libs)
    or
    ./gradlew server (This will run the server)

However, importing into IDE and running it from there is totally different thing, as there are some proto related build path changes to be done on GRPC and changing the compiler levels to 1.8 for few modules etc.

Thanks and Regards,
Mahesh Yaddanapudi.

**Reply by maheshyaddanapudi** [reply]
Another curious question.

Did you clone the repository or download it as a zip ?

I cloned it and when I tried to remove the .git folder and run build, I faced the above exception too. 

I'm not entirely sure though.

**Reply by AditiRitam1209** [reply]
use  git clone and upgrade to java 11 . it will work

**Reply by SonicBrother** [reply]
it worked

**Comment by apanicker-nflx** [comment, marked as answer]
We recommend using the gradle wrapper and gradle versions packaged alongwith the application to build and run the server. Further, please ensure that you are using java11 as your jvm version.
